starter-code/src/functions-and-arrays.js

```diff
@@ -11,14 +18,40 @@ var words = [
   'crackpot'
 ];
 
+function findLongestWord(a){
+  var index = 0;
+  for(i=1; i<a.length; i++){
+    if(a[i].length>a[index].length){
+       index = i;
+    }
+  }
+  return a[index]; 
+}
+
 // Calculating a Sum
 
 var numbers = [6, 12, 1, 18, 13, 16, 2, 1, 8, 10];
+function sumArray(a){
+  if(a[0]!=null){
+    const reducer = (accumulator, currentValue) => accumulator + currentValue;
+    return a.reduce(reducer);
```

Comment:
Good job!! You are using ECMAscript 6 (we will see it on week 4). If you feel comfortable, use it. If not, you can use syntax we are using at class.
You can use reduce directly:
```function sumArray(a){
  if(a[0]!=null){
    return a.reduce((accumulator, currentValue) => accumulator + currentValue);
  }
  return 0;
}